Data/tasks.js

```diff
@@ -169,49 +169,49 @@ const tasks = {
   work: [
     {
       task: 'Work is money, money is luxury, let life be luxury?',
-      id: 31,
+      id: 24,
       seen: false,
```

Comment:
Between different categories, can we have a gap of 100? So that if we need to add data to a category, we can add without changing all subsequent ID's?

Reply:
between ID's of different categories. 
like if the last ID of first category is 24, the first ID of the next category should start at 124? or even 100